lib/app.dart

```diff
@@ -29,7 +29,7 @@ import 'package:fokus/pages/caregiver/child_dashboard_page.dart';
 import 'package:fokus/pages/caregiver/award_form_page.dart';
 import 'package:fokus/pages/caregiver/badge_form_page.dart';
 import 'package:fokus/pages/caregiver/panel_page.dart';
-import 'package:fokus/pages/caregiver/plan_details_page.dart';
+import 'file:///D:/Fokus/fokus/lib/pages/plan_details_page.dart';
```

Comment:
Heeere we go again!

Reply:
![obraz](https://user-images.githubusercontent.com/36575365/92786308-6b641d80-f3a8-11ea-95a9-f7528b4740c7.png)